integration-cli/docker_cli_help_test.go

```diff
@@ -133,26 +126,17 @@ func (s *DockerSuite) TestHelpTextVerify(c *check.C) {
 			helpCmd := exec.Command(dockerBinary, args...)
 			helpCmd.Env = newEnvs
 			out, stderr, ec, err = runCommandWithStdoutStderr(helpCmd)
-			if len(stderr) != 0 {
-				c.Fatalf("Error on %q help. non-empty stderr:%q", cmd, stderr)
-			}
-			if strings.HasSuffix(out, "\n\n") {
-				c.Fatalf("Should not have blank line on %q\nout:%q", cmd, out)
-			}
-			if !strings.Contains(out, "--help=false") {
-				c.Fatalf("Should show full usage on %q\nout:%q", cmd, out)
-			}
-			if err != nil || ec != 0 {
-				c.Fatalf("Error on %q help: %s\nexit code:%d", cmd, out, ec)
-			}
+			c.Assert(len(stderr), checker.Equals, 0, check.Commentf("Error on %q help. non-empty stderr:%q", cmd, stderr))
+			c.Assert(out, checker.Not(checker.HasSuffix), "\n\n", check.Commentf("Should not have blank line on %q\nout:%q", cmd, out))
+			c.Assert(out, checker.Contains, "--help=false", check.Commentf("Should show full usage on %q\nout:%q", cmd, out))
```

Comment:
No need to print `out` here, it will be printed out in case of failure :
```go
c.Assert(out, checker.Contains, "--help=false", check.Commentf("Should show full usage on %q", cmd))
```